projects.go

```diff
@@ -1286,6 +1287,7 @@ type ChangeApprovalConfigurationOptions struct {
 	ResetApprovalsOnPush                      *bool `url:"reset_approvals_on_push,omitempty" json:"reset_approvals_on_push,omitempty"`
 	DisableOverridingApproversPerMergeRequest *bool `url:"disable_overriding_approvers_per_merge_request,omitempty" json:"disable_overriding_approvers_per_merge_request,omitempty"`
 	MergeRequestsAuthorApproval               *bool `url:"merge_requests_author_approval,omitempty" json:"merge_requests_author_approval,omitempty"`
+	MergeRequestsDisableCommittersApproval    *bool `url:"merge_requests_disable_committers_approval" json:"merge_requests_disable_committers_approval,omitempty"`
```

Comment:
You missed `omitempty` for the `url` tag.

Reply:
Bah, thank you.  I actually missed it on the `json` tag too, but the test suite at least caught that one for me...